pkg/registry/apis/folders/register.go

```diff
@@ -175,6 +145,13 @@ func (b *FolderAPIBuilder) UpdateAPIGroupInfo(apiGroupInfo *genericapiserver.API
 			return err
 		}
 
+		legacyStore := &legacyStorage{
```

Comment:
Just moved this here as we don't need legacy storage if we are not creating dualWriter